docs/blocks/block-events.md

```diff
@@ -283,7 +284,13 @@ Removes the current block and replaces it with the defined block in the same pos
 ```json
 "wiki:place_block": {
   "set_block": {
-      "block_type": "minecraft:grass"
```

Comment:
Please make this a separate codeblock and subheader. That way we can have a normal `set_block` and one with block states